man/en/sos-report.1

```diff
@@ -14,7 +14,7 @@ sos report \- Collect and package diagnostic and support data
           [--preset preset] [--add-preset add_preset]\fR
           [--del-preset del_preset] [--desc description]\fR
           [--batch] [--build] [--debug] [--dry-run]\fR
-          [--label label] [--case-id id]\fR
+          [--estimate-mode] [--label label] [--case-id id]\fR
```

Comment:
Since we are not generating any usable collection, perhaps `--estimate-only` here? Just to be incredibly clear users should not expect an archive.

Reply:
Yes, that was the term I was looking for.